3-UsingAPIs/Week1/homework/ex2-checkDoubleDigits.js

```diff
@@ -12,8 +12,14 @@ Complete the function called `checkDoubleDigits` such that:
   "Expected a double digit number but got `number`", where `number` is the 
   number that was passed as an argument.
 ------------------------------------------------------------------------------*/
-function checkDoubleDigits(/* TODO add parameter(s) here */) {
-  // TODO complete this function
+function checkDoubleDigits(number) {
+  return new Promise((resolve, reject) => {
+    if (number > 9 && number < 100) {
```

Comment:
```suggestion
    if (number >= 10 && number <= 99) {
```
code style: for me it's easier to read when you're using the `>=` and `<=` operators